src/components/categoryPage/subcategoryFilter/subcategoryFilter.tsx

```
@@ -16,77 +23,170 @@
+          break;
+
+        case 'ArrowUp':
+          event.preventDefault();
+          setHighlightedIndex((prev) =>
+            prev <= 0 ? allSubcategories.length - 1 : prev - 1
+          );
+          break;
+
+        case 'Tab':
+          event.preventDefault();
+          const isShiftTab = event.shiftKey;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)